src/code/z_onepointdemo.c

```diff
@@ -947,7 +948,7 @@ s32 OnePointCutscene_SetInfo(PlayState* play, s16 subCamId, s16 csId, Actor* act
 
         case 4220:
             csInfo->keyFrames = (player->actor.world.pos.z < -15.0f) ? D_80122C3C : D_80122C64;
-            csInfo->keyFrameCnt = 1;
+            csInfo->keyFrameCount = ARRAY_COUNT(D_80122C3C);
```

Comment:
Questionable to me if array_count of one of the ternary branches is better than just "1" :shrug: